test/support/utils_sycl.h

```diff
@@ -79,6 +79,11 @@ auto async_handler = [](sycl::exception_list ex_list) {
     }
 };
 
+//Check that type is device copyable including types which are deprecated in sycl 2020
+template <typename T>
+static constexpr bool check_device_copyable_w_deprecrated_sycl2020 =
```

Comment:
```suggestion
static constexpr bool check_if_device_copyable_by_sycl2020_or_by_old_definition =
```

But really, we should stop relying on the old definition and testing for it, sooner rather than later.

Reply:
I definitely agree that we should take care of this soon.  
I'll take this suggestion.